environments/icds-new/inventory.ini

```diff
@@ -66,14 +66,15 @@ pgucrstandby1
 pgbouncer_max_connections=3500
 pgbouncer_default_pool=775
 postgresql_max_connections=800
-postgresql_work_mem=32MB
-postgresql_shared_buffers=8GB
-postgresql_effective_cache_size=24GB
+postgresql_work_mem=3495kB
+postgresql_shared_buffers=16GB
+postgresql_effective_cache_size=48GB
 postgresql_checkpoint_segments=128
 postgresql_checkpoint_completion_target=0.9
 postgresql_maintenance_work_mem = 2GB
 postgresql_wal_buffers=16MB
 postgresql_default_statistics_target = 500
+postgresql_max_wal_size = 1GB
```

Comment:
may want to consider upping this again. still getting

```
checkpoints are occurring too frequently (27 seconds apart)",,"Consider increasing the configuration parameter ""max_wal_size"".",,,,,,,""
```

up from 4 seconds

Reply:
Turns out this is the default. I'll bump it to 5GB.

Reply:
https://github.com/dimagi/commcare-cloud/pull/2029/files#diff-0688e9dafa291b12fa97b468ab69dfa1R77

Reply:
Which machine was that warning on? I think it is fine to increase it, but I think based on the volume of data we load nightly into the UCR machine, we will likely see it no matter how large the size is. That was my takeaway when I looked into this for the warehouse machine. Although, they also suggested I set it to the full ram of the machine, so I would not worry about this configuration, but maybe also not worry about those messages.

Reply:
I'm not too concerned about the warning itself, more about the performance implications of it.